Makefile

```diff
@@ -2,6 +2,7 @@ PACKAGE = easyrdf
 VERSION = $(shell php -r "print json_decode(file_get_contents('composer.json'))->version ?? 'dev';")
 distdir = $(PACKAGE)-$(VERSION)
 PHP = $(shell which php)
+PHP_FLAGS = -d memory_limit=1G
```

Comment:
I was wondering what memory requirements your new approach has. Is 1 GB only for tests here?

Reply:
Yeah, the memory increase is to make sure there is enough memory for the "large RDF/XML file" test to run.

There is no increase in memory requirement, unless you want to load large files.